### PR TITLE
feat(settings): add clearLocalStorage action

### PR DESCRIPTION
**Pull request by AlexZakablukov**
**What this PR does** 📖
implement clear localstorage and indexed db feature by clicking on "clear local storage" button in settings

**Which issue(s) this PR fixes** 🔨
[AP-700](https://satellite-im.atlassian.net/browse/AP-700)

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by molimauro — APPROVED** [review]

**Review by WanderingHogan — APPROVED** [review]

**Comment by WanderingHogan** [comment]
@AlexZakablukov can you add a window refresh or something to reload the page after they click the clear local storage button

File: locales/en-US.js [hunk 1]
```diff
@@ -473,6 +474,9 @@ export default {
       mailbox_manager_not_found: 'Mailbox manager not found',
       mailbox_manager_not_initialized: 'Mailbox manager not initialized',
     },
+    storage: {
+      database_not_cleared: 'Could not clear database',
```
Comment (stavares843) [inline locales/en-US.js:478]: what would trigger this error btw? I tried clear storage and then clear storage again but always shows ''Local storage cleared successfully''

**Comment by stavares843** [comment]
will add a ticket for the reload

**Comment by stavares843** [comment]
[AP-751](https://satellite-im.atlassian.net/browse/AP-751)

**Comment by stavares843** [comment]
[AP-752](https://satellite-im.atlassian.net/browse/AP-752)

**Pull request merged**